packages/core/src/components/ProgressCard.tsx

```diff
@@ -17,16 +17,17 @@
 import React, { FC } from 'react';
 import { makeStyles } from '@material-ui/core';
 
-import InfoCard from '../layout/InfoCard';
+import { InfoCard } from '@backstage/core';
```

Comment:
Can't import from where you already are :grin:

Reply:
Could you explain a bit more? I don't get these nuances.